src/themes/default/list.ts

```diff
@@ -1,37 +1,42 @@
-export default (theme, { getTextStyle }) => {
+import { BasisTheme, ListTheme, ThemeHelpers } from "../types";
+
+export default (
+  theme: Pick<BasisTheme, "colors" | "space" | "radii" | "fontWeights">,
+  { getTextStyle }: ThemeHelpers
+): ListTheme => {
   return {
-    getCSS: ({ targetElement, type, variant, textStyle }) => {
-      switch (targetElement) {
+    getCSS: (options) => {
```

Comment:
Destructuring the param seems to be a better and nicer code

Reply:
Can't destructured it, some of the properties are only available to some `targetElement`. see: https://github.com/moroshko/basis/pull/247/files#diff-9bdb7564c7c64bf70f00b13cc604c7e53f8994437029037ea3ae38671fabfc9fR236

As such typescript would give errors as not all target elements have all the properties